src/main/java/com/exadel/movie_new/controller/MovieController.java

```diff
@@ -0,0 +1,48 @@
+package com.exadel.movie_new.controller;
+
+import com.exadel.movie_new.model.Movie;
+import com.exadel.movie_new.service.MovieService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@RestController @RequestMapping("/movies")
+public class MovieController {
+
+    private MovieService movieService;
+
+    public MovieController(MovieService movieService) {
+        this.movieService = movieService;
+    }
+
+
+    @PostMapping("/")

```

Comment:
You not need any mapping. It must be simple @POST. And in case you need add pathParam - use @GET @Path("/{movieId}")